README.rst

```diff
@@ -10,6 +10,9 @@ PyGMT
 .. image:: http://img.shields.io/pypi/v/pygmt.svg?style=flat-square
     :alt: Latest version on PyPI
     :target: https://pypi.python.org/pypi/pygmt
+.. image:: https://anaconda.org/conda-forge/pygmt/badges/installer/conda.svg
+    :alt: Latest version on conda-forge
+    :target: https://conda.anaconda.org/conda-forge
```

Comment:
Maybe point to the PyGMT conda package page at https://anaconda.org/conda-forge/pygmt

```suggestion
    :target: https://anaconda.org/conda-forge/pygmt
```